app/controllers/FileChecksResultsController.scala

```diff
@@ -20,16 +19,14 @@ class FileChecksResultsController @Inject() (
     val graphqlConfiguration: GraphQLConfiguration,
     val consignmentService: ConsignmentService,
     val consignmentStatusService: ConsignmentStatusService,
-    val frontEndInfoConfiguration: FrontEndInfoConfiguration,
-    val configuration: Config
+    val frontEndInfoConfiguration: ApplicationConfig,
+    val applicationConfig: ApplicationConfig
```

Comment:
You can remove the old one `frontEndInfoConfiguration`.

Reply:
Sorry, I was rushing. I've removed this and updated the tests.